<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitCMake</class>
 <widget class="QDialog" name="InitCMake">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>440</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>InitCMake</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_ProjectName"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEdit_ProjectPath"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_ProjectPath">
     <property name="text">
      <string>项目根目录</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_ProjectName">
     <property name="text">
      <string>项目名称</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton_ProjectPath">
     <property name="text">
      <string>选取目录</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_Init">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
